google/resource_container_cluster.go

```diff
@@ -149,6 +149,21 @@ func resourceContainerCluster() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
 
+			"legacy_abac": {
```

Comment:
Are there any other parameters here? Should we simplify this for the users into a simple toggle?
`enable_legacy_abac` ?